src/bundlingCalls/bundleExecutor.ts

```diff
@@ -202,7 +202,8 @@ export class BundleExecutor {
       return ret;
     }
 
-    if (!(bundleId in this._timers) && this._options.delayThreshold > 0) {
+    if (!(bundleId in this._timers) && this._options.delayThreshold! > 0) {
+      //@ts-ignore
```

Comment:
Let's not use `@ts-ignore` if possible (here and everywhere).

Reply:
Sure, removed unnecessary `@ts-ignore`